components/studio/projects/fixtures/projects_templates.json

```diff
@@ -29,13 +29,13 @@
         },
         "environments": {
           "Jupyter Lab": {
-            "repository": "ghcr.io/scaleoutsystems/stackn",
-            "image": "jupyter-stackn:v0.6.0-beta.1",
+            "repository": "ghcr.io/scilifelabdatacentre",
+            "image": "serve-jupyterlab:latest",
```

Comment:
Are you sure about using `latest` and not specific versions?

Reply:
In our workflow the latest versions of images under our control will have been already tested and verified to work in serve so using latest tag should be ok here.

Reply:
I agree with Johan. This is a design thing that we've decided upon. But we also talked about using the tag production. This can easily be modified so perhaps we should keep in for now.